school/rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date, :book
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+  end
```

Comment:
- [ ] After creating this rental we need to add it to the respective rental arrays of the book and person, as a rental cannot exist independent of the two objects.

  ```
  def initialize(date, book, person)
    @date = date
    
    @book = book
    book.rentals << self
    
    @person = person
    person.rentals << self
  end
  ```